libraries-bom-table-generation/helpers/javaModulesNamePretty.yaml

```diff
@@ -0,0 +1,174 @@
+google-cloud-bom: "Google Cloud Bill of Materials"
```

Comment:
Another intermediary file.

Reply:
I've combined the metadata from this file with the product link metadata as a temporary workaround while we work on updating the artifacts in Maven Central with this metadata.